main/lua-xml/APKBUILD

```diff
@@ -17,28 +17,28 @@ for _v in $_luaversions; do
 	subpackages="$subpackages lua$_v-xml:split_${_v/./_}"
 done
 
-source="http://viremo.eludi.net/LuaXML/LuaXML_$pkgver.zip
-0001-Fix-error-where-TAG-should-be-xml.TAG.patch"
-
-_builddir="$srcdir"
+source="$pkgname-$pkgver.tar.gz::https://github.com/LuaDist/luaxml/archive/$pkgver.tar.gz
+	0001-Fix-error-where-TAG-should-be-xml.TAG.patch"
 
+builddir="$srcdir"
```

Comment:
This should point to the root of the unpacked package, so  `$srcdir/$pkgname-$pkgver`.